website/content/en/preview/_index.md

```diff
@@ -19,11 +18,30 @@ Karpenter works by:
 * **Scheduling** the pods to run on the new nodes
 * **Removing** the nodes when the nodes are no longer needed
 
-As a cluster administrator, you can configure an unconstrained Karpenter provisioner when it is first installed and not change it again.
-Other times, you might continue to tweak the provisioner or create multiple provisioners for a cluster used by different teams.
-On-going cluster administrator tasks include upgrading and decommissioning nodes.
+As someone using Karpenter, once your Kubernetes cluster and the Karpenter controller are up and running (see [Getting Started]({{<ref "./getting-started" >}}), you can:
 
-As an application developer, you can make specific requests for capacity and features you want from the nodes running your pods.
-Karpenter is designed to quickly create the best possible nodes to meet those needs and schedule the pods to run on them.
+* **Set up provisioners**: By applying a provisioner to Karpenter, you can configure constraints on node provisioning and set timeout values for node expiry.
+
+* **Deploy workloads**: When deploying workloads, you can request that scheduling constraints be met to direct which nodes Karpenter provisions for those workloads.
+
+Karpenter supports Kubernetes scheduling constraints, including Kubernetes well-known labels, annotations, and taints.
+It also supports constraints specific to the cloud provider (such as AWS).
+The following Kubernetes and cloud provider constraints are supported by Karpenter at the provisioner, node, and pod levels:
+
+* Provisioner Level
+  - [Zones](https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone)
+  - [Capacity Types]({{<ref "./provisioner" >}}),
+  - [Instance Types](https://kubernetes.io/docs/reference/labels-annotations-taints/#nodekubernetesioinstance-type)
+  - [Taints](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
+* Node Level
```

Comment:
I'm not sure this Node section belongs, they're derived from the set of all available instance types plus the requirements set on the provisioner.  They're not something separate that users can configure.